docs/guide/web-simulation.md

```diff
@@ -324,6 +324,68 @@ This folder also contains a `server.sh` utility script to automatically start an
 
 Please make sure that the `WEBOTS_HOME` variable is set before running the simulation and session server scripts.
 
+#### Using Docker
+
+The simulation server can also be run in a Docker container.
+In this section we provide a sample Docker setup that could be used to run the simulation on localhost.
+
+You can use the following `Dockerfile` to build your Docker image.
+```
+FROM cyberbotics/webots:R2021a-ubuntu18.04
```

Comment:
This image is not available yet as Webots R2021a is not yet released.
To avoid having to maintain this we can use instead `FROM cyberbotics/webots:latest`

Reply:
Which is the Ubuntu version of `cyberbotics/webots:latest`?

Reply:
Oh good point, it is 20.04, then we have to use `R2020b-rev1-ubuntu18.04` and maintain it :-/

Reply:
Or even better:
```suggestion
FROM cyberbotics/webots:{{ webots.version.package }}-ubuntu18.04
```

If it works in code parts.